Tella/Utils/Validator.swift

```diff
@@ -74,6 +74,23 @@ extension String {
         }
         return true
     }
-
+    func validateWithRegex(pattern: String) -> Bool {
+        guard !self.isEmpty else {
+            return false
+        }
+        guard validateRegex(value: self, pattern: pattern) else {
+            return false
+        }
+        return true
+    }
+    func uwaziLoginValidtor() -> Bool {
```

Comment:
I think we have already this validator which is `textValidator` you can use it instead